### ISSUE TITLE
Port it to SeaMonkey

### ISSUE THREAD

**Issue by tonymec** [Boldewyn/digraphs#18]
It ought to be possible to fix this together with issue #12 (Port it to Thunderbird) and/or #13 (Port it to Fennec) but I'm creating the issue so that (if ever this extension gets any support again, of which I'm far from hopeful since "simple" issues one year old are still unattended) SeaMonkey doesn't get forgotten.
I would expect this addon to work in Thunderbird and SeaMonkey after adding either one additional targetApplication tag for each app, or one for toolkit@mozilla.org, or both.
See http://forums.mozillazine.org/viewtopic.php?f=19&t=1355415 for details.

**Comment by tonymec** [comment]
See also http://forums.mozillazine.org/viewtopic.php?f=19&t=500361&p=11873777#p11873777 and any replies to it.

**Comment by Boldewyn** [comment]
Well, actually the addon is actively used (by me, that is), but my problem is that the current implementation hooks into the DOM of the current page and does thus not work in XUL elements. I am in the middle of experimenting to move it to a Jetpack extension, but I haven't had much time to do so in the last three months.

**Comment by tonymec** [comment]
Yes, but that's what blocks issue #10. Working in XUL elements is orthogonal with typing Unicode in mail messages (in Thunderbird or SeaMonkey) or in text entry forms in webpages (in SeaMonkey). I still don't see anything (other than mentioning the apps in install.rdf) which would prevent the extension to work "as-is" (in content) in Thunderbird, SeaMonkey, or indeed in any Toolkit application.

**Comment by tonymec** [comment]
Current release supports all three of Firefox, Thunderbird and SeaMonkey. Actually it works better in SeaMonkey than in Firefox AFAICT (both at Gecko/Toolkit 47.0a1).

**Comment by tonymec** [comment]
Oops! Wrong repository.
